org.gnome.meld.json

```diff
@@ -0,0 +1,94 @@
+{
+    "app-id": "org.gnome.meld",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "meld",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a",
+        "*.pyc",
+        "*.pyo"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "rename-appdata-file": "meld.appdata.xml",
+    "rename-desktop-file": "meld.desktop",
+    "rename-icon": "meld",
+    "finish-args": [
+        /* X11 + XShm */
+        "--share=ipc", "--socket=x11",
+        /* Wayland */
+        "--socket=wayland",
+        /* Filesystem */
+        "--filesystem=host",
+        /* dconf */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [{
+            "name": "gtksourceview",
+            "build-options": {
+                "config-opts": [
+                    "--enable-vala=no"
+                ]
+            },
+            "sources": [{
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.6.tar.xz",
+                "sha256": "7aa6bdfebcdc73a763dddeaa42f190c40835e6f8495bb9eb8f78587e2577c188"
+            }]
+        },
+        {
+            "name": "pycairo",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python2 setup.py install --prefix=${FLATPAK_DEST}"
+            ],
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://github.com/pygobject/pycairo/releases/download/v1.15.4/pycairo-1.15.4.tar.gz",
+                    "sha256": "ee4c3068c048230e5ce74bb8994a024711129bde1af1d76e3276c7acd81c4357"
+                }
+            ]
+        },
+        {
+            "name": "pygobject",
+            "build-options": {
+                "env": {
+                    "PYTHON": "python2"
```

Comment:
I believe meld uses Python 3 now?

Reply:
Yeah, didn't saw that :+1:

Reply:
No you should *remove* the pycairo and pygobject builds then, it is already in the runtime.

Reply:
Done;-)

Reply:
Also I am trusting you verified it does still run? =)